<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_AddSupplier</class>
 <widget class="QFrame" name="F_AddSupplier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutMain">
     <item>
      <layout class="QVBoxLayout" name="layoutLeftSide">
       <item>
        <widget class="QLabel" name="L_company">
         <property name="text">
          <string>Company *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_company">
         <property name="maxLength">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_name">
         <property name="text">
          <string>Name *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_name">
         <property name="maxLength">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_vtaNumber">
         <property name="text">
          <string>VTA Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_vtaNumber"/>
       </item>
       <item>
        <widget class="QLabel" name="L_email">
         <property name="text">
          <string>Email Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_email">
         <property name="maxLength">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_phone">
         <property name="text">
          <string>Phone *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_phone">
         <property name="maxLength">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_address">
         <property name="text">
          <string>Address *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_address">
         <property name="maxLength">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_city">
         <property name="text">
          <string>City *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_city">
         <property name="maxLength">
          <number>50</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_state">
         <property name="text">
          <string>State</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_state">
         <property name="maxLength">
          <number>50</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutRightSide">
       <item>
        <widget class="QLabel" name="L_postalCode">
         <property name="text">
          <string>Postal Code</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Le_postalCode"/>
       </item>
       <item>
        <widget class="QLabel" name="L_country">
         <property name="text">
          <string>Country</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_country"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Bt_addSupplier">
         <property name="text">
          <string>Add Supplier</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
